features/facebookImport/src/components/dataStructureMiniStory/dataStructureMiniStory.jsx

```diff
@@ -15,12 +15,24 @@ const DataStructureMiniStory = ({ data }) => {
         return b.value - a.value;
     });
 
+    let totalFiles = 0;
+    data.forEach((a) => (totalFiles += a.count));
```

Comment:
Complete nit: While this works, it's an interesting case for `reduce` to make a one liner:

```
const totalFiles = data.reduce((previous, current) => previous + current.count, 0);
```